### ISSUE TITLE
WallpaperPluginException when trying to build

### ISSUE THREAD

**Issue by pareta2** [rocksdanister/lively#1845]
**Describe the bug**
I am C#.net developer, I want to do some customization to this lively plugin and I followed this piece for developer setup
https://github.com/rocksdanister/lively/wiki/Building

but I am getting this error everytime, also attaching the log
![image](https://github.com/rocksdanister/lively/assets/8460598/1b47d0af-bf22-4636-b3bc-746d5cdfe7b8)

Basically I am seeing the error 
"2023-08-17 16:26:36.3552|ERROR|Lively.Core.WinDesktopCore|Failed launching wallpaper: Process exited before giving HWND."
for which I have followed the link https://github.com/rocksdanister/lively/issues/1614
My antivirus is not deleting any file and Visual C++ Redistributable is already installed as part of visual studio 2022

**To Reproduce**
1. Run the code all projects will compile
2. Run lively.exe from debug folder
3. Run Lively.UI.WinUI project and change wallpaper
![image](https://github.com/rocksdanister/lively/assets/8460598/2920dbf8-0441-477e-9ee3-74800459a06b)

**Expected behavior**
Expectation is to change the wallpaper

**Screenshots/Video**
If applicable, add screenshots/video to help explain your problem.
![image](https://github.com/rocksdanister/lively/assets/8460598/d1f3f8cc-a26d-4614-adb4-858c732afb56)

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10 1909]
- Which wallpaper was running at the time if any. 
- Default windows wallpaper is running

2023-08-17 16:25:50.5250|INFO|Lively.App|
Lively v2.0.7.0 x86 desktop-native en-US
OS: Microsoft Windows 11 Home Single Language 10.0.22621
CPU: Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz
GPU: NVIDIA GeForce GTX 1650 Ti
GPU: Intel(R) UHD Graphics

**Additional context**

**Log file (Important)**
Share the diagnostic files by:

Open Lively settings (Gear icon) top-right -> System -> Log File -> Save
or 
[lively_log_20230817_165822.zip](https://github.com/rocksdanister/lively/files/12368885/lively_log_20230817_165822.zip)

Navigate to Appdata folder `C:\Users\<UserName>\AppData\Local\Lively Wallpaper\logs`

**Comment by rocksdanister** [comment]
For webpage wallpapers `../Plugins/Cef` should contain CefSharp build:
https://github.com/rocksdanister/lively-cef

For video wallpapers `../Plugins/Mpv` should contain mpv build.

**Comment by pareta2** [comment]
Hello @rocksdanister 
Thank you for your response, can you send the link of MPV also and let me try again.

**Comment by rocksdanister** [comment]
Its there in the Building page.

**Comment by pareta2** [comment]
Hello @rocksdanister 

I just checked your livelycef link and I am already using yours cef version

![image](https://github.com/rocksdanister/lively/assets/8460598/9caa1418-812d-4f51-b131-3b123cea0e5b)

**Comment by rocksdanister** [comment]
No need to ping me.

There are other files in the build, not just exe?

**Comment by pareta2** [comment]
Hi I tried placing all the file along with exe but error is same.
![image](https://github.com/rocksdanister/lively/assets/8460598/6f5f8ee7-0f18-492b-b258-a7d8bde95f8b)

Cef folder 
![image](https://github.com/rocksdanister/lively/assets/8460598/38e7fdd5-65d2-4227-93ec-a3ed7b0e436b)

MPV folder
![image](https://github.com/rocksdanister/lively/assets/8460598/dd7bb24a-12ed-466e-86ec-0d7724a597df)

**Comment by pareta2** [comment]
[lively_log_20230818_120833.zip](https://github.com/rocksdanister/lively/files/12376235/lively_log_20230818_120833.zip)

**Comment by rocksdanister** [comment]
I would try copying the Plugins folder from official Lively installer to the visual studio build folder and verify.

**Comment by pareta2** [comment]
I copied the Plugins folder and code is working now, so may I know what part I am missing when compiling Lively respective projects and placing all the files as you stated for Plugin folder?

**Comment by rocksdanister** [comment]
It's upto you to figure out, I don't know what went wrong in your system when you build cefsharp.. one guess the build was debug instead of release.
